src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -925,6 +925,13 @@ constexpr void DataBox<tmpl::list<Tags...>>::merge_old_box(
        '0')...};
 }
 
+// Silence "maybe-uninitialized" warning for GCC-6. The warning only occurs in
+// Release mode.
+// Note that clang also defines `__GNUC__`, so we need to exclude it.
```

Comment:
can this warning pop really not be scoped more than this?

Reply:
Do you mean move it out of `DataBox.hpp`? I can try..

Reply:
I moved it to the two linear solver tests that emit the warning. It didn't work to just wrap the problematic function call in the pragmas, but I had to also wrap the include of `DataBox.hpp`. Do you like this approach better than wrapping the problematic constructor in `DataBox.hpp`?

What I really don't understand is why the issue occurs in `CG/Test_ElementActions.cpp` and `GMRES/Test_ResidualMonitorActions.cpp`, but NOT in `GMRES/Test_ElementActions.cpp` and `CG/Test_ResidualMonitorActions.cpp`, which are basically the same...

Reply:
Sorry, I meant putting it around the one line that causes the error, not the whole constructor. Is that not possible?

Reply:
ok that works, too.